packages/components/src/DateInput/DateInputWithIconButton/DateInputWithIconButton.module.scss

```
@@ -0,0 +1,52 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/component-library/styles/animation";
```

This is not available in KAIO

```suggestion
@import "../../../styles/animation";
```

---

Nice catch. Completely missed it.